lib/generators/active_fedora/noid/templates/migrations/create_minter_states.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMinterStates < ActiveRecord::Migration
```

Comment:
I don't think we should continue to copy bad patterns when there are better alternatives